src/Components/AwayScoreForm.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+const AwayScoreForm = props => {
+    const [inputValue, setInputValue] = useState();
+
+
+    const checkInput = input => props.setAwayScore( input < 0 ? 0 : input)
```

Comment:
Great job knowing how to break this off and have it perform the logic you needed! Very clean.